src/main/java/fi/metatavu/metamind/bot/functions/BeerApiListBeersMetaBotFunction.java

```diff
@@ -0,0 +1,133 @@
+package fi.metatavu.metamind.bot.functions;
+
+import java.util.Collections;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.Collections'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)